### PR TITLE
chore(docs) Adding pandas to third-party tool

### PR DESCRIPTION
**Pull request by amyshwang**
The officla Pytthon client page containt the main info so this is more like a pointer...

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -77,12 +77,13 @@ pages for the following topics:
...
-- [Prometheus](/docs/third-party-tools/prometheus) monitoring and alerting.
-- [Grafana](/docs/third-party-tools/grafana) instructions for connecting QuestDB
+- [Grafana](/docs/third-party-tools/grafana/) instructions for connecting QuestDB
   as a datasource for building visualizations and dashboards
-- [Kafka](/docs/third-party-tools/kafka/overview) guide for ingesting data from
+- [Kafka](/docs/third-party-tools/kafka/overview/) guide for ingesting data from
   Kafka topics into QuestDB
-- [Telegraf](/docs/third-party-tools/telegraf) guide for collecting system
+- [Pandas](/docs/third-party-tools/pandas/) for ingesting Pandas DataFrames via ILP
+- [Prometheus](/docs/third-party-tools/prometheus/) monitoring and alerting.
```
Comment (amunra) [inline docs/introduction.md:85]: Why some lines end with '.' and others don't?

File: docs/third-party-tools/pandas.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+if __name__ == '__main__':
+    example()
+```
+
+## See also
+
+For detailed documentation, please see:
+
+- [Sender.dataframe()](https://py-questdb-client.readthedocs.io/en/latest/api.html#questdb.ingress.Sender.dataframe)
```
Comment (amunra) [inline docs/third-party-tools/pandas.md:68]: Can you add a link to all the dataframe examples?

https://py-questdb-client.readthedocs.io/en/latest/examples.html#data-frames

**Review by amunra — APPROVED** [review]
Thanks!

**Review by amunra — CHANGES_REQUESTED** [review]
Sorry.. hit the wrong button.

A few minor changes required.

**Review by amunra — CHANGES_REQUESTED** [review]
Minor tidy-up.

File: docs/third-party-tools/pandas.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+description:
+  Guide for using Pandas with QuestDB via the official QuestDB Python client
+  library
+---
+
+[Pandas](https://pandas.pydata.org/) is a fast, powerful, flexible, and
+easy-to-use open-source data analysis and manipulation tool, built on top of the
+Python programming language. The
+[QuestDB Python client](https://py-questdb-client.readthedocs.io/en/latest/index.html)
+provides native support for ingesting Pandas DataFrame via the InfluxDB Line Protocol (ILP).
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/third-party-tools/pandas.md:12]:
```suggestion
provides native support for ingesting Pandas dataframes via the InfluxDB Line Protocol (ILP).
```

File: docs/third-party-tools/pandas.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+## Prerequisites
+
+- QuestDB must be running and accessible. You can do so from
+  [Docker](/docs/get-started/docker), the
+  [binaries](/docs/get-started/binaries), or
+  [Homebrew](/docs/get-started/homebrew) for macOS users.
+- Python 3.7 or later
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/third-party-tools/pandas.md:20]:
```suggestion
- Python 3.8 or later
```

File: docs/introduction.md [hunk 5]
```diff
@@ -77,12 +77,13 @@ pages for the following topics:
...
 
-- [Prometheus](/docs/third-party-tools/prometheus) monitoring and alerting.
-- [Grafana](/docs/third-party-tools/grafana) instructions for connecting QuestDB
+- [Grafana](/docs/third-party-tools/grafana/) instructions for connecting QuestDB
   as a datasource for building visualizations and dashboards
-- [Kafka](/docs/third-party-tools/kafka/overview) guide for ingesting data from
+- [Kafka](/docs/third-party-tools/kafka/overview/) guide for ingesting data from
   Kafka topics into QuestDB
-- [Telegraf](/docs/third-party-tools/telegraf) guide for collecting system
+- [Pandas](/docs/third-party-tools/pandas/) for ingesting Pandas DataFrames via ILP
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/introduction.md:84]:
```suggestion
- [Pandas](/docs/third-party-tools/pandas/) for ingesting Pandas dataframes via ILP
```

**Review by amunra — APPROVED** [review]
Lookin' good now.

**Pull request merged**